### ISSUE TITLE
Integrating with Winston.js

### ISSUE THREAD

**Issue by danilodeveloper** [bluejamesbond/Scribe.js#45]
Hello!
There is a way to integrate with Winston.js? For example using streaming, transport or something else? If not, I think I can send a PR to do it.

I'm asking this because currently I'm using Winston.js in my apps, so I would like to reduce the impact of switch to Scribe.

Anyway, Scribe.js looks like awesome! Tks!!

**Comment by guillaumewuip** [comment]
Scribe is very modular so you probably can make bridges between Scribe modules (Console2, WebPanel) and Winston. There isn't a real API for that but I think it's possible. Feel free to send a PR, it'll be welcome :)

You actually want to see Winston logs in the Scribe.js Webpanel ?

**Comment by danilodeveloper** [comment]
Hi @guillaumewuip!!
Yes, I'd like to see Winston logs in the Scribe.js webpanel. Probably we can do it creating a [Winston custom transport](https://github.com/winstonjs/winston#adding-custom-transports)

**Comment by guillaumewuip** [comment]
Yes, for now the WebPanel find the logs store locally in files. Maybe you could stores Winston logs with the LogWriter module and the WebPanel should do his job fine :)

**Comment by lzilioli** [comment]
@danilodeveloper any luck with this? I'm looking for the same functionality.
